docs/guides/revsets.md

```diff
@@ -0,0 +1,20 @@
+# Configuring `revsets.log`
+
+## Explaining the default log revset
+
```

Comment:
I would at least include something here about what this is, rather than leaving it blank. I know the motivation further down is about large repositories, but at least a short sentence describing the actual topic at hand first would be good.